### DISCUSSION TITLE
Add youtube search of songs

### DISCUSSION THREAD

**Discussion by kozaka-tv** [kozaka-tv/Rocksmith-Servant#215] [Ideas]
Would be nice, to have a modul, what do a search on youtube according to the request (or setlist), and delivers back a youtube url for the video of the song.

**Comment by skully** [comment]
Examples for low-budget search only: https://gitlab.com/skullyka/junkfooder/-/blob/master/plugins/youtube.py?ref_type=heads
https://gitlab.com/skullyka/bitbot/-/blob/master/src/plugins/yt.py?ref_type=heads
Creates a complete playlist from the setlist, but needs oauth authentication: https://gitlab.com/skullyka/yt-plyalist-creator
Also don't forget youtube data api quota (10k daily quota limit): https://developers.google.com/youtube/v3/determine_quota_cost


If it has something in the bracket like (bass), it returns something other than the song, like a tutorial video for the song on bass.
